index.html

```diff
@@ -0,0 +1,259 @@
+            the industry's standard</p>
+        <div class="botones3">
+            <bUtton type="button" class="r">html</bUtton>
+            <button type="button" class="o">bootstrap</button>
+            <button type="button" class="g">Ruby</button>
+        </div>
+        <div class="brian">
+            <button type="button" class="see">See Proyect</button>
+        </div>
+    </div>
+    <div class="ad">
+        <img class="profesional" src="pictures/profesionalart.png" alt="Snow">
+        <h3 class="dashbord">Profesional Art Printing Data</h3>
+        <p class="texto3">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+            the industry's standard</p>
+        <div class="botones3">
+            <bUtton type="button" class="r">html</bUtton>
+            <button type="button" class="o">bootstrap</button>
+            <button type="button" class="g">Ruby</button>
+        </div>
+        <div class="brian">
+            <button type="button" class="see">See Proyect</button>
+        </div>
+    </div>
+    <div class="ria">
+        <img class="profesional" src="pictures/profesionalart.png" alt="Snow">
+        <h3 class="dashbord">Profesional Art Printing Data</h3>
+        <p class="texto3">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+            the industry's standard</p>
+        <div class="botones3">
+            <bUtton type="button" class="r">html</bUtton>
+            <button type="button" class="o">bootstrap</button>
+            <button type="button" class="g">Ruby</button>
+        </div>
+        <div class="brian">
+            <button type="button" class="see">See Proyect</button>
+        </div>
+    </div>
+    <div class="ald">
+        <img class="profesional" src="pictures/profesionalart.png" alt="Snow">
+        <h3 class="dashbord">Profesional Art Printing Data</h3>
+        <p class="texto3">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+            the industry's standard</p>
+        <div class="botones3">
+            <bUtton type="button" class="r">html</bUtton>
+            <button type="button" class="o">bootstrap</button>
+            <button type="button" class="g">Ruby</button>
+        </div>
+        <div class="brian">
+            <button type="button" class="see">See Proyect</button>
+        </div>
+    </div>
+
+    <div class="z">
+        <h2 class="p">About me</h2>
+        <p class="hello">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.</p>
+        <section class="abouted">
+            <div class="proyect2">
+                <button type="button" class="see">Get My Resume</button>
+            </div>
```

Comment:
- [ ] As you can see in the screenshots below, your project miss the background image in the `About me` section, Kindly work on adding this background image

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/154772691-dac62692-52b4-45d9-b8d1-f2e70a06579d.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/154772694-fa10e1f0-7cea-4e4d-ab51-99c6642472c5.png)

Reply:
Resolved 👍